software/README.md

```diff
@@ -4,115 +4,103 @@ This is an example set of workloads that uses the SHA3 rocc accelerator.
 
 # Getting Started
 
-Since SHA3 accelerator uses custom instruction not supported by the vanilla toolchain, 
-these workloads require a modified version of Spike simulator to build, which can be found
-[here](https://github.com/ucb-bar/esp-isa-sim). If you are using Chipyard, make sure to source
-`env.sh` (or if you also have vanilla toolchain installed, `env-esp-tools.sh`). Otherwise, make sure 
-the Spike executable is in your `PATH`.
+Since SHA3 accelerator uses custom instruction not supported by the vanilla
+toolchain, these workloads require a modified version of Spike simulator to
+build, which can be found [here](https://github.com/ucb-bar/esp-isa-sim). If
+you are using Chipyard, you should install the esp-tools toolchain which comes
+with the appropriate version of Spike.
```

Comment:
Just to correct another error in the old README: The workloads can be built with the regular toolchain since the RoCC instructions are encoded with the `.insn` directive.

```suggestion
Running these workloads require a modified Spike simulator that supports the SHA3
custom extension, which can be found [here](https://github.com/ucb-bar/esp-isa-sim). If
you are using Chipyard, you should install the esp-tools toolchain which comes
with the appropriate version of Spike.
```